packages/wonder-blocks-form/components/choice-internal.js

```diff
@@ -57,8 +56,7 @@ type Props = {|
  * and RadioGroup. This design allows for more explicit prop typing. For
  * example, we can make onChange a required prop on Checkbox but not on Choice
  * (because for Choice, that prop would be auto-populated by CheckboxGroup).
- */
-export default class ChoiceInternal extends React.Component<Props> {
+ */ export default class ChoiceInternal extends React.Component<Props> {
```

Comment:
😱It's always this file that prettier messes up.

Reply:
👻